src/checkpoint/checkpoint_stats.c

```diff
@@ -44,6 +63,38 @@ __wt_checkpoint_set_handle_stats(
     WT_STAT_CONN_SET(session, checkpoint_handle_skip_duration, ckpt->handle_stats.skip_time);
 }
 
+/*
+ * __wt_checkpoint_stats_time_update --
+ *     Update time-related stats.
+ */
+void
+__wt_checkpoint_stats_time_update(WT_SESSION_IMPL *session)
+{
+    WT_CONNECTION_IMPL *conn;
+    WT_CKPT_CONNECTION *ckpt;
```

Comment:
Same here.
Actually, instead can you just have `WT_CKPT_CONNECTION` and initialize it to `ckpt = &S2C(session)->ckpt`. I don't see anyplace in the function actually using `conn`.

Reply:
Updated in d4dce6e7d0dd719e8eb5c24fe5b9f138555f7a82